components/ActionSheet/docs/typical-use.md

```diff
@@ -0,0 +1,41 @@
+### Typical use
+
+Create an instance of `MDCActionSheetController` and add actions to it, once actions are added. You can now 
+present the action sheet controller.
+
+<!--<div class="material-code-render" markdown="1">-->
+#### Swift
+```swift
+let actionSheet = MDCActionSheetController(title: "Action Sheet", 
+                                         message: "Secondary line text")
```

Comment:
[stylenit] Indentation alignment of message with the pervious params.

Reply:
Done